actor-tests/src/test/java/org/apache/pekko/util/JavaConverterScala3InlineTest.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * license agreements; and to You under the Apache License, version 2.0:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * This file is part of the Apache Pekko project, which was derived from Akka.
+ */
+
+package org.apache.pekko.util;
+
+import java.time.Duration;
+import java.util.*;
+import java.util.concurrent.CompletableFuture;
+
+import scala.Option;
+import scala.concurrent.Future;
+
+/**
+ * These tests are here to ensure that methods from {@link org.apache.pekko.util.FutureConverters},
+ * {@link org.apache.pekko.util.JavaDurationConverters} and {@link
+ * org.apache.pekko.util.OptionConverters} for use within Java can be compiled from with Java
+ * sources. This is because methods marked with the Scala 3 inline keyword cannot be called from
+ * within Java (see https://github.com/lampepfl/dotty/issues/19346)
+ */
+public class JavaConverterScala3InlineTest {
+  public void Test() {
```

Comment:
@Test

Reply:
The point of the rest isn't to validate that the methods are correct but rather that the `*.java` compiles.

Reply:
shouldn't method names start with lower case chars? - so `Test()` doesn't seem like a good method name